cluster/manifests/roles/scalyr-key-rotation-rbac.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scalyr-key-rotation
+  namespace: visibility
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  resourceNames:
+  - scalyr-keys
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scalyr-key-rotation
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  resourceNames:
+  - scalyr-delegation-token
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: scalyr-key-rotation
+  namespace: visibility
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: scalyr-key-rotation
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: zalando-iam:zalando:service:scalyr-key-rotation
```

Comment:
If you intend to use PlatformCredentialsSets, this would be `stups_scalyr-key-rotation`.

Reply:
With the deprecation of `stups` will this also be renamed at some point? I've tripped over this issue a couple of times in different places now

Reply:
Very unlikely, because it's used everywhere.